code/modules/clothing/shoes/magboots.dm

```diff
@@ -53,4 +53,25 @@
 obj/item/clothing/shoes/magboots/syndie/advance //For the Syndicate Strike Team
 	desc = "Reverse-engineered magboots that appear to be based on an advanced model, as they have a lighter magnetic pull. Property of Gorlex Marauders."
 	name = "advanced blood-red magboots"
-	slowdown_active = SHOES_SLOWDOWN
\ No newline at end of file
+	slowdown_active = SHOES_SLOWDOWN
+
+/obj/item/clothing/shoes/magboots/clown
+	desc = "The prankster's standard-issue clowning shoes. Damn they're huge! There's a red light on the side."
+	name = "clown shoes"
+	icon_state = "clown"
+	item_state = "clown_shoes"
+	slowdown = SHOES_SLOWDOWN+1
+	item_color = "clown"
+	var/footstep = 1	//used for squeeks whilst walking

```

Comment:
Does anything use this variable? If the actual clown shoes do, then their code does not automatically apply to this.

Reply:
The comment was from the clown shoes. I believed that the var was called from some other file checking to make sounds when the player moves, but commenting it out does not seem to have any effects, on either my shoes or the original clown shoes. The line could probably be removed from the original shoes aswell.